### ISSUE TITLE
Add an argument to choose between appending or prepending aggregated values in with_total()

### ISSUE THREAD

**Issue by alixdamman** [larray-project/larray#694]
@gdementen I noticed in one of the in-house models that an argument to prepend new total values would be helpful.

**Comment by gdementen** [comment]
Unsure about this. I would say it depends how often they do this. I fear they will be no end to this: they will want to append for one axis but prepend on another. They can always reindex, right?

**Comment by alixdamman** [comment]
Yes but reindexing takes computer time. We could first give the possibility to either prepend or append for all axes and groups. But it's, once again, adding a new argument to a method and increase the complexity of LArray which is right now quite high.

**Comment by gdementen** [comment]
adding before and after arguments like in .insert **might** be a better idea than adding a specific argument. 
**edit**: Hmmm, maybe not given than it's potentially for all axes at once while insert is only on one axis at a time.
